quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -29,15 +30,21 @@ public static PropertyMapper[] getMappers() {
                         .paramLabel("<handler>")
                         .build(),
                 fromOption(LoggingOptions.LOG_CONSOLE_OUTPUT)
+                        .isEnabled(LoggingPropertyMappers::isConsoleEnabled)
```

Comment:
> since these are always specified together it may make sense to just have a single method for this:

Yep, that'd be better :)

> A string contstant could be extracted for the messages, or there could even be a holder or interface that would directly associate the message with the boolean function.

Do we want to create any more comprehensive logic around it? I like the flexibility in that as there might be several different use-cases such as obtaining value from specific property `Configuration.getKcConfigValue("xyz") == "abc"` even though there might be provided some duplications. 

But if you think it'd be better, I can do it.

Reply:
Or we can provide some sort of hybrid approach similar to the DeprecatedMetadata as we could potentially in the future enhance only the metadata around enablement. I'll try to write some prototype.

Reply:
I tried to create EnabledMetadata instead of that, but I don't like it much. I'll merge only the builder methods for now and then we can discuss.

https://github.com/mabartos/keycloak/commit/d402d817aab28c121b68aa95f9a0a1cd833cdb70